logging/net_appender_test.go

```diff
@@ -0,0 +1,151 @@
+package logging
```

Comment:
These tests were copied from `web/server/net_logger_test.go`. Just setting up the logger in terms of the new API.

Note I didn't copy two tests. One was about logger recursion and the other about differing log levels. Because those parameters from the "new net logger" api were removed, the tests are no longer relevant.